spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/AvroBinaryIterator.java

```diff
@@ -73,6 +82,11 @@ public boolean hasNext() {
   @Override
   public InternalRow next() {
     try {
+      // TODO(nileshny) : confirm if this is correct way to get number of Avro rows parsed
+      try {
+        numberOfRowsParsed = Math.addExact(numberOfRowsParsed, 1);
```

Comment:
why Math.addExact?

Reply:
to detect overflow

Reply:
I replaced int with long as that's appropriate type in this case.